src/compiler/app-core/bundle-app-core.ts

```diff
@@ -12,13 +12,14 @@ export const generateRollupOutput = async (
     return null;
   }
 
-  const { output } = await build.generate(options);
-  return output.map((chunk) => {
+  const { output }: { output: [OutputChunk, ...(OutputChunk | OutputAsset)[]] } = await build.generate(options);
+  return output.map((chunk: OutputChunk | OutputAsset) => {
```

Comment:
For my own understanding, what the difference between these two - Chunk and Asset?